lib/phoenix_live_view/helpers.ex

```diff
@@ -478,6 +481,20 @@ defmodule Phoenix.LiveView.Helpers do
     EEx.compile_string(expr, options)
   end
 
+  @doc """
+  Provides `~H` sigil with HTML safe Live HEEx syntax inside source files.
```

Comment:
We need to add some basic docs about the features of ~H sigils and the big disclaimer you need Elixir v1.12.